post.js

```diff
@@ -0,0 +1,7 @@
+class Post {
+  constructor(body){
+    this.body = body;
+  }
+}
+
```

Comment:
I guess this model wasn't finished but naturally no reason to have a class without being able to return the body